ctf-gamemode/ctf_rendering.lua

```diff
@@ -36,7 +36,7 @@ AddEventHandler('onClientResourceStart', function(resourceName)
     -- https://docs.fivem.net/docs/game-references/text-formatting/#content-formatting-codes
     AddTextEntry("textRenderingEntry", "~a~")
     -- Request the texture dictionary for our instructional buttons
-    RequestStreamedTextureDict("commonmenutu")
+    RequestStreamedTextureDict("commonmenutu", false)
```

Comment:
Second parameter is `bPriority`, which ORs a specific request flag, setting it to false means no stream load priority. In the original example it had no priority anyway. 

Setting it to `true` puts the request on a priority list so it gets loaded before anything else.

Reply:
"I know, but since no argument was put, it's false by default?